src/main/java/com/gorlah/kappabot/integration/discord/DiscordBot.java

```diff
@@ -50,19 +47,8 @@ public void onMessageReceived(MessageReceivedEvent event) {
             return;
         }
 
-        event.getChannel().sendMessage(processMessageEvent(event)).queue();
-    }
-
-    private String processMessageEvent(MessageReceivedEvent event) {
-        return formatResponse(functionProcessor.process(new DiscordRequestMetadata(event)), event);
-    }
-
-    private String formatResponse(String response, MessageReceivedEvent event) {
-        Map<String, String> values = new HashMap<>();
-
-        values.put("user.name", event.getMessage().getAuthor().getName());
-        values.put("user.mention", event.getMessage().getAuthor().getAsMention());
-
-        return StringSubstitutor.replace(response, values);
+        event.getChannel()
+                .sendMessage(functionProcessor.process(new DiscordRequestMetadata(event)))
+                .queue();
```

Comment:
@ploober, how do you like this class now? 😉

Reply:
:+1: